js/src/utils/concatenateListOfWords.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * A function that joins a list of words using the sepators provided as the second and third argument.
+ *
+ * Example:
+ *
+ * concatenateListOfWords(['A', 'B', 'C']) will return 'A, B and C'
+ *
+ * @param  {Array<string>} listWords A list of words to concatenate.
+ * @param  {string} [separator] The separator between words.
+ * @param  {string} [separatorLastElement] The separator for the last word.
```

Comment:
💅 It would be good to also indicate the default values for the optional arguments.